### ISSUE TITLE
Error running rule requireDollarBeforejQueryAssignment

### ISSUE THREAD

**Issue by purplecones** [AtomLinter/linter-jscs#275]
### Version
4.0.4
Atom: 1.8.0
OS X: 10.11.3

### Issue
I keep getting this message on line 1 of the file.

![context_js_ __users_mirzajoldic_projects_capfundr-app_and_new_issue_ _atomlinter_linter-jscs_and_resolvers_js_ __users_mirzajoldic_projects_models-graphql](https://cloud.githubusercontent.com/assets/136654/16420955/4def63aa-3d21-11e6-8337-87b551c34397.png)

```
<linter-message-line><span class="badge badge-flexible">internalError</span> Error running rule requireDollarBeforejQueryAssignment: This is an issue with JSCS and not your codebase.
Please file an issue (with the stack trace below) at: https://github.com/jscs-dev/node-jscs/issues/new
TypeError: Cannot read property 'name' of undefined
    at /Users/mirzajoldic/.atom/packages/linter-jscs/node_modules/jscs/lib/rules/require-dollar-before-jquery-assignment.js:142:30
    at Array.forEach (native)
    at Object.<anonymous> (/Users/mirzajoldic/.atom/packages/linter-jscs/node_modules/jscs/lib/rules/require-dollar-before-jquery-assignment.js:139:23)
    at Array.forEach (native)
    at Object.JsFile.iterateNodesByType (/Users/mirzajoldic/.atom/packages/linter-jscs/node_modules/jscs/lib/js-file.js:331:42)
    at Object.module.exports.check (/Users/mirzajoldic/.atom/packages/linter-jscs/node_modules/jscs/lib/rules/require-dollar-before-jquery-assignment.js:72:14)
    at [object Object].<anonymous> (/Users/mirzajoldic/.atom/packages/linter-jscs/node_modules/jscs/lib/string-checker.js:189:22)
    at Array.forEach (native)
    at [object Object].StringChecker._checkJsFile (/Users/mirzajoldic/.atom/packages/linter-jscs/node_modules/jscs/lib/string-checker.js:185:31)
    at [object Object].StringChecker.checkString (/Users/mirzajoldic/.atom/packages/linter-jscs/node_modules/jscs/lib/string-checker.js:92:14)
    at Object.lint (/Users/mirzajoldic/.atom/packages/linter-jscs/src/linter-jscs.js:207:14)
    at promises.push.Promise.then._this.emitter.emit.linter (/Users/mirzajoldic/.atom/packages/linter/lib/linter-registry.js:63:26)
    at /Users/mirzajoldic/.atom/packages/linter/lib/linter-registry.js:62:23
    at Set.forEach (native)
    at LinterRegistry.lint (/Users/mirzajoldic/.atom/packages/linter/lib/linter-registry.js:60:18)
    at /Users/mirzajoldic/.atom/packages/linter/lib/linter.coffee:98:16
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at /Users/mirzajoldic/.atom/packages/linter/lib/editor-linter.js:86:22
    at /Users/mirzajoldic/.atom/packages/linter/lib/helpers.coffee:25:18</anonymous></anonymous></linter-message-line>
```

**Comment by Arcanemagus** [comment]
Can you please read and follow the instructions in the error you posted? This isn't a bug in `linter-jscs` but in the implementation of `requireDollarBeforejQueryAssignment`.

**Comment by purplecones** [comment]
Thanks. I did but ended up posting in the wrong repo..

**Comment by purplecones** [comment]
@Arcanemagus please merge PR https://github.com/AtomLinter/linter-jscs/pull/278 to resolve this issue

**Comment by Arcanemagus** [comment]
Published in v4.0.5 😉.
